pages/press.md

```diff
@@ -836,6 +836,10 @@ include talk.html
 
 ## Mentions of Solid in Articles
 
+### 2021
+
+2021-04-15 [Interview: Web inventor Tim Berners-Lee](https://www.sciencefocus.com/future-technology/interview-web-inventor-tim-berners-lee-thinks-his-creation-is-out-of-control-heres-his-plan-to-save-it/)
```

Comment:
```suggestion
2021-04-15 [Interview: Web inventor Tim Berners-Lee thinks his creation is out of control. Here’s his plan to save it](https://www.sciencefocus.com/future-technology/interview-web-inventor-tim-berners-lee-thinks-his-creation-is-out-of-control-heres-his-plan-to-save-it/)
```

Reply:
What do you think of this suggestion, to have the full title of the article as a link ?

Reply:
Yes please, accept suggest on behalf?

Also, mind an opinion on how many characters before an ellipsis IF text overflow? (Mobile related)
 
Notetoself: Figureout overflow-x makeellipsis (Mobile related)

Reply:
For now, I think we can try avoiding ellipsis altogether: even on mobile, the text should just span on multiple lines. Let me know if this does not work in some contexts.